src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx

```diff
@@ -75,7 +75,20 @@ Here are some details on how to accomplish specific use cases:
         Collect logs from apps running in Docker containers, either standalone or when using Kubernetes
       </td>
       <td>
-        Use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/) or our [infrastructure monitoring agent with Docker](/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-container-infrastructure-monitoring/#requirements).
+        Use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/) for Kubernetes environments. 
+        For Docker environments customers have multiple options:
+        
+        * Use our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context).
+        * Forward Docker logs using our infrastructure agent. In the config file be sure to tail the `/var/log/containers/*.log` directory.
+          **Example:**
+
+          ```yml
+          logs:
+           - name: Docker-logs
+             file: /var/log/containers/*.log # Path to Docker log files
+          ```
+        * Use our [Standalone Docker image](https://hub.docker.com/r/newrelic/newrelic-fluentbit-output).
+
```

Comment:
```suggestion
        For Kubernetes environments, use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/).
        
        For Docker environments, we offer a few options:
        * Use our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context)
        * Use our infrastructure agent. In the `logging.yml` config file be sure to tail the `/var/log/containers/*.log` directory.
          **Example:**

          ```yml
          logs:
           - name: Docker-logs
             file: /var/log/containers/*.log # Path to Docker log files
          ```
        * Use our [Standalone Docker image](https://hub.docker.com/r/newrelic/newrelic-fluentbit-output).

```